app/assets/stylesheets/_footer.scss

```diff
@@ -1,38 +1,23 @@
 .footer__container {
-  position: relative;
+  position: absolute;
   bottom: 0;
   left: 0;
   height: 45px;
   width: 100%;
+  margin: 0 auto;
+  background-color: $grey-light;
```

Comment:
Yep, but I added it because it was being overridden on the profile page for some reason. I can try to track down the reason.